jac-uk/admin
# #665 - show 2d questions and answers

## What's included?
'schedule-2-d' questions and answers were not being displayed to admins. 
This must have been accidentally removed at some point, i've now inserted the fields into the application overview.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Configure an exercise to ask schedule-2-d questions. (eligibility > schedule)
- [Apply side] apply/edit application to answer schedule-2-d questions
- ensure the answers are displayed on application overview


## Risk - how likely is this to impact other areas?
🟢 Low risk - this is a self-contained bug fix

## Additional context
![image](https://user-images.githubusercontent.com/44227249/143607085-9b88ec4c-4ff1-41a4-a2da-00fb4817189d.png)


---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
